src/sinks/util/batch.rs

```diff
@@ -109,12 +111,15 @@ where
     ///
     /// This is based on the uncompressed size of the batched events, before they are
     /// serialized / compressed.
+    #[configurable(metadata(docs::type_unit = "bytes"))]
     pub max_bytes: Option<usize>,
 
-    /// The maximum size of a batch, in events, before it is flushed.
+    /// The maximum size of a batch before it is flushed.
+    #[configurable(metadata(docs::type_unit = "events"))]
     pub max_events: Option<usize>,
 
-    /// The maximum age of a batch, in seconds, before it is flushed.
+    /// The maximum age of a batch before it is flushed.
+    #[configurable(metadata(docs::type_unit = "seconds"))]
     pub timeout_secs: Option<f64>,
```

Comment:
TODO: asked about this offline. Seems like we would want each sink's defaults in the cue docs.
But how to do that with the current design ...

Reply:
Solved this here:

https://github.com/vectordotdev/vector/pull/16108